src/features/ping/index.ts

```diff
@@ -51,10 +78,9 @@ export class PingFeature extends Feature {
         }
 
         // If the message triggers dril content...
-        if (joinedMessage === "drilme") {
+        if (joinedMessage === "drilme" || joinedMessage === ":dril:") {
```

Comment:
Hm I reckon there's probably a better way to do this, since not every server will have this emoji in it. I'll look into it myself...